app.py

```diff
@@ -1,12 +1,21 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+from flask import request
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/add_Item', methods=['POST'])
+def add_Item():
+    item = request.form['title']
+    session.add_item(item)
+
+    return 'Added Successfully!'
```

Comment:
You haven't done the final step of the exercise before the stretch goals - "Once the item has been added to the list, redirect the user back to the index page again and check that the list updates as expected"

So instead of returning a string here, try returning a `redirect` with the correct arguments.